lib/Dependency.js

```diff
@@ -62,6 +62,8 @@ class Dependency {
 		this.optional = false;
 		/** @type {DependencyLocation} */
 		this.loc = undefined;
+		/** @type {string[][] | null} */
+		this.referencedExportNames = null;
```

Comment:
This should not be here. Every info should be passed via `getReferencedExports` and not via magic properties.